cpp/src/arrow/util/async_generator_test.cc

```diff
@@ -79,20 +79,14 @@ class TrackingGenerator {
   std::shared_ptr<State> state_;
 };
 
+// Iterator<TestInt> VectorIt(std::vector<TestInt> v)
```

Comment:
nit: did you mean to place this here?

Reply:
Whoops, that's embarrassing.  Thanks for catching it.  I've removed the comment.